packages/grafana-schema/src/raw/playlist/x/playlist.gen.ts

```diff
@@ -43,14 +30,6 @@ export interface Playlist {
      *  dashboards behind the tag will be added to the playlist.
      */
     value: string;
-    /**
-     * Title is the human-readable identifier for the playlist item.
-     */
-    title: string;
```

Comment:
Removing the title here is the one *real* change -- this means we will drop title before returning it rather than just ignoring it.

The other removed fields here are just an issue with sql denormalization bleeding into the public DTO